ibm-mq-metrics/src/integrationTest/java/io/opentelemetry/ibm/mq/integration/tests/WMQMonitorIntegrationTest.java

```diff
@@ -281,4 +284,34 @@ void test_otlphttp() throws Exception {
     // reads a value from the heartbeat gauge
     assertThat(metricNames).contains("ibm.mq.heartbeat");
   }
+
+  @Test
+  void test_bad_connection() throws Exception {
+    logger.info("\n\n\n\n\n\nRunning test: test_bad_connection");
+    String configFile = getConfigFile("conf/test-bad-config.yml");
+
+    ConfigWrapper config = ConfigWrapper.parse(configFile);
+    Meter meter = otelTesting.getOpenTelemetry().getMeter("opentelemetry.io/mq");
+    TestWMQMonitor monitor = new TestWMQMonitor(config, meter, service);
+    monitor.runTest();
+
+    List<MetricData> data = otelTesting.getMetrics();
+
+    assertThat(data).isNotEmpty();
+    assertThat(data).hasSize(2);
+
+    SumData<LongPointData> connectionErrors = null;
+    for (MetricData metricData : data) {
+      if ("ibm.mq.connection.errors".equals(metricData.getName())) {
+        connectionErrors = (SumData<LongPointData>) metricData.getData();
+      }
+    }
+
+    assertThat(connectionErrors).isNotNull();
+
+    LongPointData metricPoint = connectionErrors.getPoints().iterator().next();
+    String value = metricPoint.getAttributes().get(AttributeKey.stringKey("error.code"));
+
+    assertThat(value).isEqualTo("2538");
```

Comment:
Would it be possible to verify the _value_ as well?

Reply:
the value is 1, we increment a counter, is that _valuable_ ?